templates/app/offline-worker.js

```diff
@@ -107,6 +107,15 @@
       });
     },
 
+    // Make requests to directories become requests to index.html
+    extendToIndex: function (request) {
+      var url = request.url;
+      if (url[url.length - 1] === '/') {
+        url += 'index.html';
+      }
+      return new Request(url, request);
```

Comment:
It might also be worth isolating the pathname within the URL string via something like `new URL(request.url).pathname`, since a URL to a file hosted by GitHub Pages can end in a search string like `?foo=bar` (even if the server doesn't actually process the string) and/or a fragment identifier like `#anchor` (although I'm not sure if that gets preserved by the *Request* object).